data/crac-file/crac-impl/src/main/java/com/farao_community/farao/data/crac_impl/remedial_action/network_action/PstSetpoint.java

```diff
@@ -47,11 +59,11 @@ public void setSetpoint(double setpoint) {
     /**
      * Change tap position of the PST pointed by the network element at the tap given at object instantiation.
      *
-     * @param network: network to modify
+     * @param network network to modify
      */
     @Override
     public void apply(Network network) {
-        PstRange pst = new PstRange(networkElement);
+        PstRange pst = new PstRange(getId(), networkElement);
```

Comment:
This is indeed strange, but we made this choice to avoid code duplication. We will address this conception issue later but for now I will keep this method like this